### PR TITLE
Addition of new docs & figures 

### PR DESCRIPTION
**Pull request by BabutaAniket**
Pull request contains below addition:

1. Updated 12 figures based on new recent version.
2. Added Document for the following: 
   - Introduction to K8s Integration, 
   - User support, 
   - Feature support, 
   - UI Support,
   - Setup guide. 
3. Made a small typing mistake correction in [app/cluster/routes.py ](https://github.com/Fiware-Community/kubernetes-in-FiwareLab/blob/main/UI/K8s/K8s_ui/microblog2/app/cluster/routes.py) file.

### CODE REVIEW

File: docs/source/feature_support.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+************************************
+Feature support (Phase-2)
```
Comment (pooja1pathak) [inline docs/source/feature_support.rst]: Remove (Phase-2)

File: docs/source/feature_support.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+************************************
+
+Below is the list of features supported on kubernetes deployed in Phase-1:
```
Comment (pooja1pathak) [inline docs/source/feature_support.rst]: Remove Phase-1

File: docs/source/UI_support.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
+************************************
+UI support to create and manage Kubernetes cluster for the user (Phase-3)
```
Comment (pooja1pathak) [inline docs/source/UI_support.rst]: Remove Phase-3

File: docs/source/K8sIntegration.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+2. **Self-Healing**: 
+This feature of kubernetes is what makes it best among others. If any pod gets deleted manually or accidentally, it is restared by kubernetes cluster. This maintains continuity with the deployed project. This feature of kuberenetes to heal a pod is referred as auto-heal. 
+
+3. **Automated Rollouts & Rollback**: This can be achieved by rolling update. Rolling updates are the default strategy to update the running version of your app. With every new update cycle the previously launched Pod is removed and in its place a  newer Pod with latest updates is launched incrementally. 
+When any change is introduced that breaks production, then there should be  a plan to roll back that change. Kubernetes and kubectl offer a simple mechanism to roll back changes to its resources such as Deployments.
+
+Support for Kubernetes will be provided in 3 Phases as:
```
Comment (pooja1pathak) [inline docs/source/K8sIntegration.rst]: Remove Phase information

File: docs/source/UI_support.rst [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+.. figure:: figures/deployment-log.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Note:** UI may change to match user requirement.
+
+**Advantages:**
+1. Easier to create and manage kubernetes cluster through GUI.
```
Comment (pooja1pathak) [inline docs/source/UI_support.rst:82]: add blank line after every point

File: docs/source/UI_support.rst [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+2. User will be provided with option to deploy Fiware GEs through GUI.
+3. User can select Fiware GEs from the catalogue and those GEs will be deployed.
+
+**Limitations:**
+1. User should be familiar with Python to run the server.
```
Comment (pooja1pathak) [inline docs/source/UI_support.rst:90]: add blank line after every point

**Comment by pooja1pathak** [comment]
After rebase and minor changes which I have commented we can merge this PR

**Comment by BabutaAniket** [comment]
I have rebased and the recommended changes have been incorporated.

**Pull request merged**